### PR TITLE
feat: custom generator templates 2

### PR DESCRIPTION
**Pull request by AndreiGusa**
### Summary
A different solution where we use a common method to render custom templates for every generator
Can be tested with this PR https://github.com/shiftsmartinc/monorepo/pull/477

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@shiftsmartinc/feathers-generators": "^5.0.9-beta.2",
+    "@shiftsmartinc/feathers-generators": "file:packages/generators",
```
Comment (AndreiGusa) [inline packages/cli/package.json:55]: just for testing purpose to be able to run on the local packages

Comment (visormatt) [reply packages/cli/package.json:55]: We'll circle back on whether or not we can keep it this way for our use-case.

File: packages/generators/src/commons.ts [hunk 2]
```diff
@@ -93,6 +96,15 @@ export interface FeathersBaseContext extends PinionContext {
    * Used for installing development dependencies during testing.
    */
   dependencyVersions?: DependencyVersions
+  /**
+   * The type of the generator specified with the 'generate' command. (e.g., service, hook, etc.)
+   */
+  generatorName: string
```
Comment (visormatt) [inline packages/generators/src/commons.ts:102]: We can likely use a better type `"service" | "hook"`

**Review by visormatt — APPROVED** [review]
Looks great, lets iterate 🚀

**Pull request merged**